whisper/README.md

```diff
@@ -19,6 +19,27 @@ Install [`ffmpeg`](https://ffmpeg.org/):
 brew install ffmpeg
 ```
 
+Next, download the Whisper PyTorch checkpoint and convert the weights to MLX format:
+
+```
+# Take the "tiny" model as an example. Note that you can also convert a local PyTorch checkpoint in OpenAI's format.
```

Comment:
Let's move this comment out as text:

convert the weights to the MLX format. For exampl, to convert the `tiny` model use:

```
python convert.py
```

Note you can also convert a local PyTorch checkpoint which is in the original OpenAI format.

Reply:
yes 👍